src/Files.App/Helpers/Win32/Win32Helper.Process.cs

```diff
@@ -126,10 +126,5 @@ public static List<Process> WhoIsLocking(string[] resources)
 
 			return processes;
 		}
-
-		public static Task<string> GetFileAssociationAsync(string filePath)
-		{
-			return GetFileAssociationAsync(filePath, true);
```

Comment:
removed this method and changed the default value of `checkDesktopFirst` to `true` in `GetDefaultFileAssociationAsync`